apps/src/lib/kits/maker/ui/SetupGuide.jsx

```diff
@@ -75,13 +114,31 @@ export default class SetupGuide extends React.Component {
       );
     }
 
+    // Experiment 'microbit' renders additional description.
+    let isMicrobit = experiments.isEnabled('microbit');
+
     if (isCodeOrgBrowser() || isChromeOS() || isWebSerial) {
       return <SetupChecklist webSerialPort={webSerialPort} />;
     }
     return (
-      <Provider store={store}>
-        <Downloads />
-      </Provider>
+      <div>
+        <h1>{applabI18n.makerSetupPageTitle()}</h1>
+        <div style={isMicrobit ? style.boardDescriptions : {}}>
+          <Description
+            {...setupGuideContent.circuitPlayground}
+            divStyle={isMicrobit ? style.descriptionFlexCard : {}}
+          />
+          {isMicrobit && (
+            <Description
+              {...setupGuideContent.microbit}
+              divStyle={style.descriptionFlexCard}
+            />
+          )}
+        </div>
```

Comment:
I found this code bock a little confusing to understand. Since "isMicrobit" it determining which view to show, I think it's better to have a high level `if (isMicrobit)` 
```
if (isMicrobit)
  ...microbit view
else 
  ...non-microbit view
```